### PR TITLE
english version revision

### PR DESCRIPTION
**Pull request by rachelzhang0922**
**What this PR does / why we need it**:

**Will this PR make the community happier**? 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**How is this PR tested**
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify)

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: docs/guide/development/develop-deviceshifu.md [hunk 1]
```diff
@@ -43,24 +43,24 @@ type ProtocolSettings struct {
 	SocketSetting *SocketSetting `json:"SocketSetting,omitempty"`
 }
 ```
-You first need to add `MQTTSetting` to `ProtocolSettings`. You also need to add the newly created settings in `CRD` here as a struct:
+First you need to add `MQTTSetting` to `ProtocolSettings`. Then you need to add the newly written settings in `CRD` as a struct:
 ```go
-// MQTTSetting defines MQTT specific settings when connecting to an EdgeDevice
+// MQTTSetting defines specific MQTT settings when connecting to an EdgeDevice
```
Comment (BtXin) [inline docs/guide/development/develop-deviceshifu.md:48]: This part is copied from codebase, let's just keep it consistent.

File: docs/guide/development/develop-deviceshifu.md [hunk 2]
```diff
@@ -74,15 +74,15 @@ type ReturnBody struct {
...
+Create a struct named `DeviceShifu`, and take `*deviceshifubase.DeviceShifuBase` as its field. Implement `DeviceShifu` interface in `deviceshifubase.go`. `DeviceShifuBase` contains skeleton code for creating and starting a `DeviceShifu`.
+Take `MQTT` for example: 
 ```go
 // DeviceShifu implemented from deviceshifuBase
 type DeviceShifu struct {
 	base *deviceshifubase.DeviceShifuBase
 }
 ```
-You can also add protocol specific structs, take `OPCUA` as e.g:
+You can also add specific protocol structs, take `OPCUA` as example:
```
Comment (BtXin) [inline docs/guide/development/develop-deviceshifu.md:85]: I would prefer `protocol specific` here.

File: docs/guide/development/develop-deviceshifu.md [hunk 3]
```diff
@@ -91,9 +91,9 @@ type DeviceShifu struct {
 	opcuaClient       *opcua.Client
 }
 ```
-Make a method to create an instance of the struct like:
+Create an instance of the struct:
```
Comment (BtXin) [inline docs/guide/development/develop-deviceshifu.md:94]: 这里其实是强调让开发者写一个用来创建这个struct的方法，所以创建method这个部分是不太能省略的。

File: docs/guide/development/develop-deviceshifu.md [hunk 4]
```diff
@@ -91,9 +91,9 @@ type DeviceShifu struct {
...
 ```go
-// New This function creates a new Device Shifu based on the configuration
+// This function creates a new Device Shifu based on the configuration
```
Comment (BtXin) [inline docs/guide/development/develop-deviceshifu.md:96]: We need the `New` here, which refers to the function name. It is go doc's convention. Besides, this part is copied from codebase, let's keep it consistent.

File: docs/guide/development/develop-deviceshifu.md [hunk 5]
```diff
@@ -105,24 +105,24 @@ if err != nil {
...
-To register the handler you can do something like:
+To register the handler you can:
 ```go
 handler := DeviceCommandHandlerMQTT{HandlerMetaData}
 mux.HandleFunc("/"+MqttDataEndpoint, handler.commandHandleFunc())
 ```
 
 
-`DeviceShifu` can also collect device telemetries specified. If you don't know what telemetry is, please take a look at this doc before proceed: https://github.com/Edgenesis/shifu/blob/main/docs/design/deviceshifu/telemetry.md
+`DeviceShifu` can also collect specific device telemetries. If you don't know what telemetry is, please check this doc before you proceed: https://github.com/Edgenesis/shifu/blob/main/docs/design/deviceshifu/telemetry.md
```
Comment (BtXin) [inline docs/guide/development/develop-deviceshifu.md:118]: `specific device telemetries` => `device specific telemetries`

**Review by BtXin — COMMENTED** [review]
Overall looks great, some minor comments need to be addressed

**Review by tomqin93 — COMMENTED** [review]
@rachelzhang0922 added 1 comment

**Review by BtXin — APPROVED** [review]
LGTM

**Review by tomqin93 — APPROVED** [review]
LGTM

**Pull request merged**